gammapy/utils/interpolation.py

```diff
@@ -219,3 +217,35 @@ def interpolate_profile(x, y, interp_scale="sqrt"):
     return ScaledRegularGridInterpolator(
         points=(x,), values=sign * y, values_scale=interp_scale
     )
+
+
+def interpolate_invalid_data_3d(data, axis_0):
```

Comment:
For now the helper function is in gammapy.utils.interpolate and could be use for any 3d map with invalid values not only background.